static/app/views/replays/details.tsx

```diff
@@ -53,11 +34,21 @@ class ReplayDetails extends AsyncView<Props, State> {
   getEndpoints(): ReturnType<AsyncView['getEndpoints']> {
     const {params, location} = this.props;
 
-    const eventView = EventView.fromLocation(location);
-    const query = eventView.getEventsAPIPayload(location);
+    const eventList = EventView.fromSavedQuery({
+      id: '',
+      name: '',
+      version: 2,
+      fields: ['timestamp', 'replayId'],
+      orderby: 'timestamp',
+      projects: [],
+      range: '14d',
+      query: `transaction:sentry-replay`,
+    });
+    eventList.additionalConditions.addFilterValues('replayId', [params.replayId]);
+    const apiPayload = eventList.getEventsAPIPayload(location);
```

Comment:
I think we should push this logic into the ReplayEvents component so that given a `replayId`, it should query for the above event list and fetch the details for each event slug.

Reply:
Disregard this